metadata_mapper/mappers/mapper.py

```diff
@@ -120,13 +120,11 @@ def map_is_shown_by(self) -> Union[str, None]:
 
     # Mapper Helpers
     def collate_subfield(self, field: str, subfield: str) -> list:
-        return [f[subfield] for f in self.source_metadata.get(field, [])]
+        return [f[subfield] for f in self.source_metadata.get(self, field, [])]
 
-    def collate_fields(self, fieldlist):
-        ''' collate multiple field values into a single list '''
+    def collate_values(self, values):
```

Comment:
I'm not sure what this is supposed to be doing. It takes a list then filters out non-strings? Is that intended? If so, list comprehension or a `filter` would be a better fit.

Reply:
collate_values() + source_metadata_values() does the same thing that collate_fields() used to do. It behaves differently depending on whether the value is a string or an array.